dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,18 @@ def load_user
     @user = current_user
   end
 
+  # GET /api/v1/users/<user_id>/school_name
+  def get_school_name
+    render json: {
+      school_name: current_user&.school
+    }
+  end
+
+  # PATCH /api/vi/users/<user_id>/update_user_info
+  def update_user_info
```

Comment:
i was a little confused about what this endpoint would do until i looked at its implementation, which tells me we should rename this method. maybe something like `last_seen_school_info_interstitial`? that's a long name, so maybe others have better suggestions 😄

Reply:
Best I can think of is something like `update_last_seen_school_info_interstitial` as well. We're updating a field with a long name, so it feels kind of inevitable :/